Codecademy/client-modules
# feat(GridForm): added hidden input functionality

### Overview







This PR adds the ability to create hidden inputs on the `GridForm` component. It's implementation adds a new type to the `packages/gamut/src/GridForm/types.ts` file and a new component to the `GridFormInputGroup`  folder that follows the same pattern with how the other Grid Form fields are implemented, but otherwise is a just tweak on how we implement the `Input` field. 

- [x] Related to JIRA ticket: GROW-2334

### Testing Instructions 

1. go to preview deployed below. 
2. go to `GridForm` section
3.  open Hidden Field  inside of canvas and submit the form
4. see that the hidden fields are part of the resolved values on form submit under the actions tab.
